cli/command/trust/info.go

```diff
@@ -101,9 +101,50 @@ func lookupTrustInfo(cli command.Cli, remote string) error {
 	}
 	printSignatures(cli, remote, signatureRows)
 
+	roleWithSigs, err := notaryRepo.ListRoles()
+	if err != nil {
+		fmt.Fprintf(cli.Out(), "Error listing signature metadata for roles: %s\n", err)
+		return err
+	}
+
+	signerRoleToKeyIDs, baseRoleToKeyIDs := getSignerAndBaseRolesWithKeyIDs(roleWithSigs)
+
+	fmt.Fprintf(cli.Out(), "\nList of signers and their KeyIDs:\n\n")
+	printRolesWithKeyIDs(cli, signerRoleToKeyIDs)
+
+	fmt.Fprintf(cli.Out(), "\nList of admins and their KeyIDs:\n\n")
+	printRolesWithKeyIDs(cli, baseRoleToKeyIDs)
+
 	return nil
 }
 
+// Extract signer keys and base keys from the list of roles
+func getSignerAndBaseRolesWithKeyIDs(roleWithSigs []client.RoleWithSignatures) (map[string][]string, map[string][]string) {
+	signerRoleToKeyIDs := make(map[string][]string)
+	baseRoleToKeyIDs := make(map[string][]string)
+
+	for _, roleWithSig := range roleWithSigs {
+		if roleWithSig.Name == trust.ReleasesRole || roleWithSig.Name == data.CanonicalSnapshotRole || roleWithSig.Name == data.CanonicalTimestampRole {
+			continue
+		}
+
+		if data.IsDelegation(roleWithSig.Name) {
+			signerRoleToKeyIDs[notaryRoleToSigner(roleWithSig.Name)] = roleWithSig.KeyIDs
+		} else {
+			baseRoleToKeyIDs[roleWithSig.Name.String()] = roleWithSig.KeyIDs
```

Comment:
I'm thinking that we should also call `notaryRoleToSigner` here and update the `owner` const to `admin` or similar.  I don't think we should expose `targets` directly to docker users.  WDYT?